com.sweethome3d.Desktop.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Sweet Home 3D
+Comment=Interior design application
+Type=Application
+Categories=Graphics;3DGraphics;Engineering;
+Exec=/app/bin/SweetHome3D %F
+MimeType=application/x-sweet-home-3d;
+Terminal=false
```

Comment:
As you intend to upstream everything, I would add some keywords here.

Reply:
Done.